go/saltpack/decrypt.go

```diff
@@ -132,7 +148,7 @@ func (ds *decryptStream) readBlock(b []byte) (n int, lastBlock bool, err error)
 	return n, false, err
 }
 
-func (ds *decryptStream) tryVisibleReceivers(hdr *EncryptionHeader, ephemeralKey BoxPublicKey) (BoxSecretKey, BoxPrecomputedSharedKey, *SymmetricKey, int, error) {
+func (ds *decryptStream) tryVisibleReceivers(hdr *EncryptionHeader, ephemeralKey BoxPublicKey) (BoxSecretKey, *SymmetricKey, int, error) {
```

Comment:
My bad, no. We still need those for decryption to try all the anonymous key boxes efficiently.